packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -62,9 +71,10 @@ export const useAuthModal = props => {
     }, [showMyAccount]);
 
     const handleSignOut = useCallback(async () => {
+        setIsSigningOut(true);
```

Comment:
Does this ever go back to `false`?

Reply:
Or do we rely on unmount for that?

Reply:
Correct, this callback will trigger a page refresh, performing the reset for us.